README.md

```diff
@@ -6,7 +6,23 @@ Release automation for pandas.
   - [  ] `TAG` in `Makefile`
   - [  ] `GH_USERNAME` in `Makefile`
 
-If running for the first time be sure to initialize repos
+## **If running for the first time:**
+
+- Create a conda enviroment based on environment.yml
+```sh
+conda env create -f environment.yml
+```
+- Activate the environment 
+```sh
+conda activate pandas-release
```

Comment:
In Linux is `source activate pandas-release`, probably worth adding it as a comment

Reply:
Nowadays `conda activate` should be the cross-platform recommended way to do this? (at least I am using conda activate on linux)

Reply:
I guess it's because I didn't do the `conda init`:
```
[mgarcia@localhost pandas]$ conda activate pandas-dev

CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.
```

On a second thought, probably better to leave this as is, and no add extra noise anyway.